lib/packages_manager.py

```diff
@@ -85,4 +85,8 @@ def discover_packages():
         LOG.error("No packages found in versions repository directory")
         raise
 
+    if not package_list:
+	LOG.error("No packages to build. Packages directory is empty")
```

Comment:
@mopsfelder We already log the stack trace for `BaseException`s in `host_os.py`, which includes the exception custom message. This will duplicate this line in the log and stdout, so it's unnecessary.